src/main/java/pages/CartPage.java

```diff
@@ -0,0 +1,126 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.List;
+
+public class CartPage extends BasePage {
+    public static final String CART_ITEM = "//*[text() = '%s']/ancestor::*[@class='cart_item']";
```

Comment:
how can I change to @FindBy here if there is '% s'?

Reply:
no way :) just as you did, define a String